src/scenes/languageCodes/LanguageCodesDetail.js

```diff
@@ -9,12 +9,18 @@ import constants from '../../redux/codes/constants';
 import { getLanguageCodeById, createLanguageCode, deprecateLanguageCode, undeprecateLanguageCode } from '../../redux/codes/actions';
 
 class LanguageCodeDetail extends Component {
+  constructor() {
```

Comment:
props?

Reply:
I'm not using them in the constructor, so they are not really necessary. Unless you consider them a "best practice"

Reply:
I consider them best practice -> 
```
The constructor for a React component is called before it is mounted. When implementing the constructor for a React.Component subclass, you should call super(props) before any other statement. Otherwise, this.props will be undefined in the constructor, which can lead to bugs.
```
https://reactjs.org/docs/react-component.html#constructor